### PR TITLE
Revert "disable upgrades to 26 on 32-bit"

### PR DESCRIPTION
**Pull request by come-nc**
Reverts #446

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
  *
  */
 
+
```
Comment (szaimen) [inline index.php:24]:
```suggestion
```

**Review by szaimen — CHANGES_REQUESTED** [review]
also you deleted the updater.phar?

**Comment by come-nc** [comment]
> also you deleted the updater.phar?

No?

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by blizzz — APPROVED** [review]

**Pull request merged**